Swagger update for 2024-07-30-preview version for recoveryservices-backup

pikumarmsft16 opened the pull request:
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

pikumarmsftnew16 committed:
Copy files from preview/2024-04-30-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

pikumarmsftnew16 committed:
Update version to preview/2024-07-30-preview

Updated the API version from preview/2024-04-30-preview to preview/2024-07-30-preview.

pikumarmsftnew16 committed:
Added tag for 2024-07-30-preview in readme file

pikumarmsftnew16 committed:
Adding HANA scaleout containery type and protectable item type related changes

jeevan-gaikwad commented on specification/recoveryservicesbackup/resource-manager/readme.md:
```diff
@@ -28,7 +28,7 @@ These are the global settings for the RecoveryServicesBackup API.
 description: Open API 2.0 Specs for Azure RecoveryServices Backup service
 openapi-type: arm
-tag: package-2024-04
+tag: package-preview-2024-07
```
We do not change this tag for preview version. 
Ref
https://github.com/Azure/azure-rest-api-specs/pull/20934/files#diff-278e6b2057b04d09991cfb9fedfda4baf53f7f073d14b022ade16da0ee7abf57

jeevan-gaikwad commented on specification/recoveryservicesbackup/resource-manager/readme.md:
````diff
@@ -37,11 +37,11 @@ license-header: MICROSOFT_MIT
 
 ```yaml $(package-passivestamp)
-tag: package-passivestamp-2023-01-15
+tag: package-preview-2024-07
````
same here

jeevan-gaikwad commented on specification/recoveryservicesbackup/resource-manager/readme.md:
````diff
@@ -37,11 +37,11 @@ license-header: MICROSOFT_MIT
 
 ```yaml $(package-activestamp)
-tag: package-2024-04
+tag: package-preview-2024-07
````
same here.

jeevan-gaikwad commented on specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/preview/2024-07-30-preview/bms.json:
```diff
@@ -10675,6 +10675,16 @@
       "x-ms-discriminator-value": "HanaHSRContainer"
     },
+    "AzureVmWorkloadSAPHanaScaleoutProtectableItem": {
+      "description": "Azure VM workload-specific protectable item representing HANA scaleout.",
```
should it be HANA **scale out**?

pikumarmsftnew16 committed:
Incorporating tag version related comments in readme file

pikumarmsftnew16 committed:
Ran prettier

pikumarmsftnew16 committed:
Ran prettier

pikumarmsftnew16 committed:
Copy files from preview/2024-04-30-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

pikumarmsftnew16 committed:
Update version to preview/2024-08-30-preview

Updated the API version from preview/2024-04-30-preview to preview/2024-08-30-preview.

pikumarmsftnew16 committed:
Added tag for 2024-08-30-preview in readme file

pikumarmsftnew16 committed:
Copy files from preview/2024-04-30-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

pikumarmsftnew16 committed:
Update version to preview/2024-08-30-preview

Updated the API version from preview/2024-04-30-preview to preview/2024-08-30-preview.

pikumarmsftnew16 committed:
Added tag for 2024-08-30-preview in readme file

pikumarmsftnew16 committed:
Copy files from preview/2024-04-30-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

pikumarmsftnew16 committed:
Update version to preview/2024-09-30-preview

Updated the API version from preview/2024-04-30-preview to preview/2024-09-30-preview.

pikumarmsftnew16 committed:
Added tag for 2024-09-30-preview in readme file

pikumarmsftnew16 committed:
Copy files from preview/2024-04-30-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

pikumarmsftnew16 committed:
Update version to preview/2024-07-30-preview

Updated the API version from preview/2024-04-30-preview to preview/2024-07-30-preview.

pikumarmsftnew16 committed:
Added tag for 2024-07-30-preview in readme file

pikumarmsftnew16 committed:
Fixing prettier check
